### PR TITLE
Release v7.27.0

### PR DESCRIPTION
**Pull request by lildude**
Time for a new release.

- [ ] Gem pushed to RubyGems & GitHub Package Registry 
- [ ] Deployed to GitHub.com
- [ ] Grammars deployed to Syntax highlighting service - highlighting won't be 💯 up-to-date until this happens

I'm aiming to make and deploy this release on the week starting 11 Sep '23 and will perform a final PR review and grammar sync just before I merge this PR.

### CODE REVIEW

**Comment by lildude** [comment]
This is now live everywhere.

File: Rakefile [hunk 1]
```diff
@@ -59,7 +59,7 @@ end
 # The error count will need to be adjusted here until such time as all grammars are 100% error free.
 desc "Check that compiling the grammars doesn't introduce any new unexpected errors"
 task :check_grammars do
-  expected_error_count = 29  # This count should only ever go down. If it goes up, there's a new issue that needs to be addressed before updating the grammar.
+  expected_error_count = 28  # This count should only ever go down. If it goes up, there's a new issue that needs to be addressed before updating the grammar.
   rm_rf "linguist-grammars"
   output, status = Open3.capture2e("script/grammar-compiler", "compile", "-o", "linguist-grammars")
```
Comment (qjoner) [inline Rakefile:64]:
```

```

File: Rakefile [hunk 2]
```diff
@@ -59,7 +59,7 @@ end
 # The error count will need to be adjusted here until such time as all grammars are 100% error free.
 desc "Check that compiling the grammars doesn't introduce any new unexpected errors"
 task :check_grammars do
-  expected_error_count = 29  # This count should only ever go down. If it goes up, there's a new issue that needs to be addressed before updating the grammar.
```
Comment (qjoner) [inline Rakefile:62]: Rakefile

Comment (qjoner) [reply Rakefile:62]:
```
- [ 
```dependabot.yml

Comment (qjoner) [inline Rakefile:62]: 29

File: grammars.yml [hunk 3]
```diff
@@ -1052,7 +1052,7 @@ vendor/grammars/turtle.tmbundle:
 - source.sparql
 - source.turtle
 vendor/grammars/typst-grammar:
```
Comment (qjoner) [inline grammars.yml:1054]: grammars.yml

**Review by qjoner — APPROVED** [review]
grammars.yml

**Pull request merged**